main/observing/observation_run.py

```diff
@@ -614,6 +637,7 @@ def _shutdown_procedure(self, calibration, cooler=True):
         """
         logging.info("Shutting down observatory.")
         time.sleep(5)
+        self.monitor.run_th_monitor = False
```

Comment:
Do NOT add this here.  _shutdown_procedure is called during observations when we temporarily close up due to weather, and you don't want to stop the monitor at those times.  Place this at the end of stop_threads() which is only called at the very very end.